controllers/bootstrap/init.go

```diff
@@ -1374,11 +1376,48 @@ func (b *Bootstrap) updateApprovalMode() error {
 		return err
 	}
 
-	if err = b.UpdateCsOpApproval(); err != nil {
+	if err = b.UpdateOpApproval("ibm-common-service-operator"); err != nil {
 		klog.Errorf("Failed to update common service operator subscription: %v", err)
 		return err
 	}
 
+	if err = b.UpdateOpApproval("operand-deployment-lifecycle-manager-app"); err != nil {
+		klog.Errorf("Failed to update operand-deployment-lifecycle-manager-app subscription: %v", err)
+		return err
+	}
+
+	if err = b.UpdateOpApproval("ibm-namespace-scope-operator"); err != nil {
+		klog.Errorf("Failed to update namespace-scope-operator subscription: %v", err)
+		return err
+	}
+
+	if err = b.UpdateOpApproval(constant.ICPOperator); err != nil {
+		if !errors.IsNotFound(err) {
+			klog.Errorf("Failed to update %s subscription: %v", constant.ICPOperator, err)
+			return err
+		}
+		klog.Infof("%s not installed, skipping", constant.ICPOperator)
```

Comment:
Also may I ask that if we could be more specific about the information?
For example, we could say `skipping updating approval strategy`.